lib/logger.js

```diff
@@ -218,19 +272,18 @@ class Logger {
 			return this.humanReadableFormatter(obj);
 		}
 
-		let formatted = {
+		const formatted = {
 			_time: this.formatTime(obj.timestamp),
 			_tags: obj.tags
 		};
 
-		formatted = Hoek.applyToDefaults(formatted, obj.data || {});
-		formatted = Hoek.applyToDefaults(formatted, obj.requestInfo || {});
+		Object.assign(formatted, obj.data || {}, obj.requestInfo || {});
```

Comment:
Does `Hoek.applyToDefaults` and `Object.assign` have the same behavior when it comes to deeply nested objects?

Reply:
applyToDefaults does a copy (i think). However i don't foresee any issue with this.